documentation/assemblies/managing/assembly-rolling-updates.adoc

```diff
@@ -0,0 +1,29 @@
+// This assembly is included in the following assemblies:
+//
+// using/assembly-management-tasks.adoc
+
+[id='assembly-rolling-updates-{context}']
+= Performing manual rolling updates of Kafka and ZooKeeper clusters
+
+Strimzi supports the use of annotations on `StatefulSet` and `Pods` resources to manually trigger a rolling update of Kafka and ZooKeeper clusters through the Cluster Operator.
+Rolling updates replace the pods of the resource with new ones, with no downtime.
```

Comment:
The downtime depends on many factors - including topic configuration etc. So I would leave that part out. I also wonder if _restart the pods_ would fit better then _replace_.

Reply:
changed to _restart_ and removed downtime.